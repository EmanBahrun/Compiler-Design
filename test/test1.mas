function mult(Integer aa, Integer bb) {
        return aa * bb;
}
function main() {
        return mult(2, 3);
}