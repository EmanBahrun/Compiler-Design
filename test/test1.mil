***Error. Variable bb does not exist
