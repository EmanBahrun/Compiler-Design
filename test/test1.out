Generated code:
func mult
. aa
= aa, $0
. bb
= bb, $1
endfunc
func main
endfunc

